pysus/ftp/databases/sim.py

```diff
@@ -6,24 +6,29 @@
 
 class SIM(Database):
     name = "SIM"
-    paths = [
+    paths = (
         Directory("/dissemin/publicos/SIM/CID10/DORES"),
         Directory("/dissemin/publicos/SIM/CID9/DORES"),
-    ]
+    )
     metadata = {
         "long_name": "Sistema de Informação sobre Mortalidade",
         "source": "http://sim.saude.gov.br",
         "description": "",
     }
-    groups = {"CID10": "DO", "CID9":"DOR"}
+    groups = {"CID10": "DO", "CID9": "DOR"}
```

Comment:
Is this information visible to the user? Because CID10 and CID9 (ICD 10 and 9, in english) are well-known to epidemiologists, DO and DOR are just meaningless acronyms, AFAIK.

Reply:
I use the acronyms to differentiate between CID9 and CID10 in the file formatting